crates/topos-tce-storage/src/store.rs

```diff
@@ -49,7 +52,18 @@ pub trait ReadStore: Send {
     /// Returns the certificates delivered by a source subnet from a position.
     fn get_source_stream_certificates_from_position(
         &self,
-        from: SourceStreamPositionKey,
+        from: CertificateSourceStreamPosition,
         limit: usize,
     ) -> Result<Vec<(CertificateDelivered, CertificateSourceStreamPosition)>, StorageError>;
+
+    fn get_target_stream_certificates_from_position(
```

Comment:
What are your thoughts on the [Rust API guidelines stance on avoiding "get_"](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter)?

Reply:
I think this API recommendation is more concerned about collections, wrappers and other simple structures. Here we have an bunch of storage `get_` APIs that usually performs complex calculations to get the data.

Reply:
I put `get_` because there is the inner action of getting the data from elsewhere.  But the whole `Store` API design can be a subject on its own.

Reply:
Let's bikeshed on this some other time.